src/openms/source/FORMAT/MSstatsFile.cpp

```diff
@@ -569,13 +539,13 @@ void OpenMS::MSstatsFile::storeISO(const OpenMS::String &filename, const Consens
           for (const OpenMS::PeptideEvidence &pep_ev : peptide_evidences)
           {
             // Write new line for each run
-            for (Size j = 0; j < consensus_feature_filenames[i].size(); j++)
+            for (Size j = 0; j < AggregatedInfo.consensus_feature_filenames[i].size(); j++)
```

Comment:
I find it a bit confusing that you are iterating over the features in the outer loop but also take the information from the AggregatedInfo datastructure.

Reply:
yes - see comment above.